frontend/packages/console-app/src/components/cloud-shell/CloudShellTab.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import CloudShellTerminal from './CloudShellTerminal';
+import CloudShellTabHeader from './CloudShellTabHeader';
+import './CloudShellTab.scss';
+
+const CloudShellTab: React.FC = () => (
+  <div className="co-cloud-shell-tab">
+    <div className="co-cloud-shell-tab__header">
+      <CloudShellTabHeader />
+    </div>
```

Comment:
I think you can simplify the header to the following and remove `CloudShellTabHeader`.
```suggestion
    <div className="co-cloud-shell-tab__header">OpenShift command line terminal</div>
```

Reply:
updated